### ISSUE TITLE
how do i make expandablecardview ?

### ISSUE THREAD

**Issue by Sankeet** [gabrielemariotti/cardslib#526]

**Comment by gabrielemariotti** [comment]
Did you check the doc?

**Comment by Sankeet** [comment]
Yes i saw it.But I am not able to expand it. I am clicking it but it doesn't expand.

**Comment by gabrielemariotti** [comment]
Here you can find all the info:
https://github.com/gabrielemariotti/cardslib/blob/master/doc/EXPAND.md

Also you can find nice examples in the demo apps.

**Comment by Sankeet** [comment]
Thanks a lot...!!
